service/javadsl/client/src/main/scala/com/lightbend/lagom/internal/javadsl/client/CircuitBreakerModule.scala

```diff
@@ -34,20 +33,7 @@ class CircuitBreakersProvider @Inject() (
   lazy val get = {
 
     // only in case some 3rd part lib are still wiring the old one.
-    system.log.warning(
-      """
-        | +----------------------------------------------------------------------------------------+
-        | | WATCH OUT!!                                                                            |
-        | | com.lightbend.lagom.internal.client.CircuitBreakers is DEPRECATED                      |
-        | | and will be removed on the next release.                                               |
-        | |                                                                                        |
-        | | Instead you should be using com.lightbend.lagom.javadsl.client.CircuitBreakersPanel.   |
-        | |                                                                                        |
-        | | If you don't understand why you are getting this warning, it's probably because        |
-        | | you are using a 3rd part library that is still wiring this deprecated CircuitBreakers. |
-        | +----------------------------------------------------------------------------------------+
-      """.stripMargin
-    )
+    system.log.warning("WATCH OUT!! - com.lightbend.lagom.internal.client.CircuitBreakers is DEPRECATED")
```

Comment:
Please, don't use capital letters or exclamation marks.

I think the original wording was great but in the refactor many information was lost:
"c.l.l.internal.client.CircuitBreakers is deprecated.n use com.lightbend.lagom.javadsl.client.CircuitBreakersPanel instead. If you don't understand why you are getting this warning, it's probably because you are using a 3rd party library that is still wiring this deprecated CircuitBreakers. Make sure to upgrade the libraries providing your service locators."